@xen-orchestra/lite/src/libs/relative-time.ts

```diff
@@ -0,0 +1,31 @@
+const DIVISIONS = [
+  { amount: 60, name: "seconds" },
+  { amount: 60, name: "minutes" },
+  { amount: 24, name: "hours" },
+  { amount: 7, name: "days" },
+  { amount: 4.34524, name: "weeks" },
+  { amount: 12, name: "months" },
+  { amount: Number.POSITIVE_INFINITY, name: "years" },
+] as const;
+
+export const toRelativeTime = (
+  from: Date | number,
+  to: Date | number,
+  locale: string
+) => {
+  const formatter = new Intl.RelativeTimeFormat(locale, {
+    numeric: "auto",
+  });
```

Comment:
`yarn run type-check` throw this error: `Property 'RelativeTimeFormat' does not exist on type 'typeof Intl'. Did you mean 'DateTimeFormat'?`
Must be fixed before merging on master

Reply:
Indeed, the fix has been sent on the Task PR.

Fixed.